war/src/main/webapp/scripts/polyfills.js

```diff
@@ -18,3 +18,13 @@ if (!Element.prototype.closest) {
 }
 
 // end closest / matches polyfill
+
+// https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#Polyfill
+if (window.NodeList && !NodeList.prototype.forEach) {
```

Comment:
haven't tested the functionality in IE 11, but this would be needed at least

Reply:
saucelabs will surely give you an open source account if your request it though it looks pretty good, so I can't imagine it failing.

Reply:
Just tested it out on a IE11 developer machine and it breaks on calls `.append` on nodes. It's not supported on IE11 https://caniuse.com/mdn-api_parentnode_append and `.appendChild` should be used instead.

Reply:
pushed (untested though)

Reply:
Now it works. No other outstanding console errors on IE11

Reply:
thanks!